tests/utils.go

```diff
@@ -3946,7 +3946,7 @@ func StartTCPServer(vmi *v1.VirtualMachineInstance, port int) {
 func StartHTTPServer(vmi *v1.VirtualMachineInstance, port int) {
 	expecter, err := LoggedInCirrosExpecter(vmi)
 	Expect(err).NotTo(HaveOccurred())
-	httpServerMaker := fmt.Sprintf("screen -d -m nc -klp %d -e echo -e \"HTTP/1.1 200 OK\\nContent-Length: 11\\n\\nHello World!\"\n", port)
+	httpServerMaker := fmt.Sprintf("screen -d -m nc -klp %d -e echo -e \"HTTP/1.1 200 OK\\nContent-Length: 12\\n\\nHello World!\"\n", port)
```

Comment:
The `\n` is not counted?

Reply:
No, the echo's input is `\"HTTP/1.1 200 OK\\nContent-Length: 12\\n\\nHello World!\"` .

Reply:
So the 12 is just for the "Hello World!"? Without the `\n` prefix?

Reply:
12 is the payload of the http packet. The `\n\n` are part of the HTTP header (it is what separates the HTTP headers from the body).